Library/Network/SPIWebSocketConnection.m

```diff
@@ -42,15 +42,15 @@ - (void)connect {
     
     // Create a new socket instance specifying the url, SPI protocol and Websocket to use.
     // The will create a TCP/IP socket connection to the provided URL and perform HTTP websocket negotiation
-    self.webSocket          = [[SRWebSocket alloc] initWithURLRequest:[NSURLRequest requestWithURL:[NSURL URLWithString:self.url] cachePolicy:NSURLRequestUseProtocolCachePolicy timeoutInterval:8.0] protocols:@[@"spi.2.3.0"]];
+    self.webSocket          = [[SRWebSocket alloc] initWithURLRequest:[NSURLRequest requestWithURL:[NSURL URLWithString:self.url] cachePolicy:NSURLRequestUseProtocolCachePolicy timeoutInterval:4.0] protocols:@[@"spi.2.3.2"]];
```

Comment:
* Read what it says, you've just changed the cache policy, not the timeout
* Protocol version doesn't change

Reply:
fixed